Scripts/RemediationScripts/AutoRemediation.ps1

```diff
@@ -129,39 +140,48 @@ Enter the choice (1/2)";
                         $log | ConvertTo-json -depth 100  | Out-File $logFile
 
                         Write-Host $([Constants]::SingleDashLine)
-                        Write-Host "Skipped remediation of failing resources of control id : [$($control.ControlId)]" -ForegroundColor $([Constants]::MessageType.Warning)
+                        Write-Host "Skipped remediation of failing resources of control id: [$($control.ControlId)]" -ForegroundColor $([Constants]::MessageType.Warning)
                         Write-Host $([Constants]::SingleDashLine)
 
                         continue;
                     }
                 }
                 Write-Host $([Constants]::SingleDashLine)
-                Write-Host "Remediating control having control id [$($control.ControlId)] using [$($control.LoadCommand)] bulk remediation script." -ForegroundColor $([Constants]::MessageType.Info)
+                Write-Host "Remediating Control Id [$($control.ControlId)] using [$($control.LoadCommand)] Bulk Remediation Script..." -ForegroundColor $([Constants]::MessageType.Info)
                 
                 [string]$timeStampString = $timestamp
                 . ("./" + "RemediationScripts\" + $control.LoadCommand)
-                $commandString = $control.InitCommand + " -SubscriptionId " +  "`'" + $SubscriptionId +  "`'" + " -RemediationType " + "DisableAllowBlobPublicAccessOnStorage" + " -Path " + "`'" + "FailedControls\" +  $SubscriptionId + ".json" + "`'" + " -AutoRemediation" + " -TimeStamp " + "`'" + $timeStampString +  "`'";
-
+                $commandString =""
+                if($control.ControlId -eq "Azure_Storage_AuthN_Dont_Allow_Anonymous"){
```

Comment:
Can we get this from some config file instead of hard coding here.

Reply:
It is only required because there are variation of command strings for different BRS. It can be reduced if variation are for let's say 5 of them out of 24 then we only need if condition for those 5